src/dataDisplay/personPanel/personPanelDetails.jsx

```diff
@@ -891,8 +896,8 @@ function PersonPanelDetails(props) {
                             {!isEmpty(selectButtonProps) && (
                                 <PersonPanelDetailsActionButton
                                     // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
Does the ESLINT suppression comment need to be moved?

Reply:
Done!

Reply:
Thanks!  Resolving.